modules/core/src/main/scala/com/caesars/tracing/NewRelic.scala

```diff
@@ -25,7 +25,14 @@ object NewRelic {
             // these are overridable using system properties if need be
             new DefaultAsyncHttpClientConfig.Builder().setUseProxySelector(true).build()
           }
-          .map(Logger(logHeaders = true, logBody = false, logAction = Some(s => ZIO.debug(s))))
+          .map(
+            Logger(
+              logHeaders = true,
+              logBody = false,
+              redactHeadersWhen = _.compareTo(CIString("Api-Key")) == 0,
+              logAction = Some(s => ZIO.logInfo(s))
```

Comment:
Info level would make this quite noisy

Reply:
That's true, but it may be a necessary evil. If things don't find their way into new relic, you really need this to troubleshoot.  Does it potentially make sense to have this be configurable from outside?

Reply:
Yes please